userguide/install.rst

```diff
@@ -139,21 +141,19 @@ On Ubuntu 17.10, there are issues with flash_tool loading the shared library 'li
     && sudo dpkg -i /tmp/libpng12.deb \
     && rm /tmp/libpng12.deb
 
-You will also need to use `the latest version of the tool <https://spflashtool.com/download/SP_Flash_Tool_v5.1744_Linux.zip>`_.
-
-4. Extract the zip files
```

Comment:
maybe the extract step is worth keeping? or reformulate the "download the latest version" above to "download and unpack..."?